README.md

```diff
@@ -223,6 +225,36 @@ app.use(function (err, req, res, next) {
 })
 ```
 
+### Verifying a token
+
+If you need to pass the token in a query string rather than an
+HTTP header, you can use the `req.verifyToken(token)` function
+provided by this middleware. This is useful when you want to
+use this library with an oauth callback.
+
+```js
+var csrf = require('csurf')
+var express = require('express')
+
+var app = express()
+app.use(csrf())
+
+app.get('/oauth_callback', function(req, res) {
+  try {
+    req.verifyToken(req.query.state); // throws
```

Comment:
Hmmm, I guess the two solve similar problems. Sometimes, it'd be helpful to be explicit, which I guess would be difficult to show in an example, but the fact that issue #43 exists means I'm not the only one that needs this use case.